.ci/ci_cache_components.sh

```diff
@@ -110,14 +118,18 @@ create_cache_asset() {
 	if [ ! -z "${check_image}" ]; then
 		if [ ! -z "${check_initrd}" ]; then
 			image_path="${kata_dir}/kata-containers-initrd.img"
-			image_name="initrd"
 		else
 			image_path="${kata_dir}/kata-containers.img"
-			image_name="image"
 		fi
+		image_name="${component_name}"
 		path=$(readlink -f "${image_path}")
 		echo $(basename "${path}") > "latest-${image_name}"
-		sudo cp "${path}" "${kata_dir}/osbuilder-${image_name}.yaml"  .
+		# ${image_name} might be decorated with build method if the image was
+		# built by dracut.  In that case, it will have a format "<image_type>.dracut".
+		# We need to get rid of the possible ".dracut" suffix to get the correct
+		# .yaml file name.
+		local image_type="$(echo ${image_name} | tr "." "\n" | head -1)"
+		sudo cp "${path}" "${kata_dir}/osbuilder-${image_type}.yaml"  .
```

Comment:
Nit: Double space before terminating period.

Reply:
Oh did I miss it?  I'm sorry - I just pushed an update which should correct this.